testsuite/features/step_definitions/command_steps.rb

```diff
@@ -328,6 +328,11 @@
   end
 end
 
+Then(/^the reposync logs should not report errors$/) do
+  result, code = $server.run('grep "ERROR:" /var/log/rhn/reposync/*.log', false)
```

Comment:
```suggestion
  result, code = $server.run('grep -H "ERROR:" /var/log/rhn/reposync/*.log', false)
```

Reply:
according to `man grep` " This is the default when there is more than one file to search." so it won't change much